src/Tests/Microsoft.Diagnostics.Monitoring.Tool.FunctionalTests/HttpApi/ApiClient.cs

```diff
@@ -334,26 +334,25 @@ public async Task<string> GetMetricsAsync(CancellationToken token)
             throw await CreateUnexpectedStatusCodeExceptionAsync(response).ConfigureAwait(false);
         }
 
-        public Task<ResponseStreamHolder> CaptureMetricsAsync(int processId, int durationSeconds, int refreshInterval, CancellationToken token)
+        public Task<ResponseStreamHolder> CaptureMetricsAsync(int processId, int durationSeconds, CancellationToken token)
         {
-            return CaptureMetricsAsync(processId, durationSeconds, refreshInterval, HttpMethod.Get, content: null, token: token);
+            return CaptureMetricsAsync(processId, durationSeconds, token: token);
```

Comment:
I think you need to add back `, HttpMethod.Get, content: null`, otherwise it is calling itself.